src/combinatorics/simple_graph/connectivity.lean

```diff
@@ -1534,16 +1613,6 @@ begin
   subst_vars
 end
 
-@[simp] lemma subgraph_of_adj_connected {v w : V} (hvw : G.adj v w) :
-  (G.subgraph_of_adj hvw).connected :=
-begin
-  split,
-  rintro ⟨a, ha⟩ ⟨b, hb⟩,
-  simp only [subgraph_of_adj_verts, set.mem_insert_iff, set.mem_singleton_iff] at ha hb,
-  obtain (rfl|rfl) := ha; obtain (rfl|rfl) := hb;
-    refl <|> { apply adj.reachable, simp },
-end
-
```

Comment:
Any reason you deleted this? (If it's because in another PR you're moving it to another file, let's not delete it here.)

Reply:
Sorry, I deleted that because I misunderstood your last comment. I have restored it now.